app/src/components/About.vue

```diff
@@ -34,7 +34,11 @@
 
         <!-- ToDo. Link to the real dGrant grant -->
         <div class="mt-16 flex flex-wrap gap-4">
-          <router-link to="/dgrants/0" @click="emitEvent('toggleAbout')">
+          <router-link
+            v-if="dgrantsGrantID && dgrantsGrantID !== 'false'"
```

Comment:
Might be cleaner to have `VITE_DGRANTS_GRANT_ID` be 0 or 1 and just check `v-if="dgrantsGrantID` and cast it to a boolean when reading the value

IME 0 or 1 is typically how boolean env vars are handled to avoid bugs where the string `false` is casted to true by JS. Plus `VITE_MAINTENANCE_MODE` is a boolean that uses 0 or 1, so this would keep things consistent

Reply:
The only issue with this is that part of the requirements is to be able to set which `grantID` actually relates to the dGrants grant. I think having the string version of `false` here should be fine, `0` is a valid `grantID` and we have no reason to cast this var to a number/boolean (in this use case at least).

Reply:
Oh of course, saw the boolean check and apparently forget we needed an actual number here 🤦

So just to confirm then, if the env var is set to zero, it will get read in as a string so the `v-if="dgrantsGrantID && dgrantsGrantID !== 'false'"` check can still resolve to true? Because if it gets read as a number, `dgrantsGrantID` would resolve to false (because `Boolean(0)` is false, `Boolean('0')` is true)

Reply:
It will be read as a string, and that check will pass!

Reply:
Thanks!